rsconf/component/sirepo.py

```diff
@@ -257,6 +258,12 @@ def _jupyterhublogin(self, z):
         self.__docker_vols.append(z.sim_api.jupyterhublogin.user_db_root_d)
         self._set_sirepo_config("sirepo_jupyterhub")
 
+    def _raydata(self):
+        if "raydata" in set(
```

Comment:
this should be a function because it is used by jupyterhublogin, too.

Reply:
4267ad6d571e91f709cadce4c3cd5ca8f6d0f58f